aws/terraform.tfvars

```diff
@@ -1,158 +1,176 @@
-####THIS section of variables are REQUIRED for 1Click Automation to run.
+######################## Terraform Variables ###################################
+
+# This file is required to succesfully deploy 1ClickECK. To customize the
+# deployment just uncomment the variable below and change its value.
+# 
+# NOTE: Your gcp cloud cli must be set with your credentials for the automation
+# to run. Be sure to run `gcloud auth login` before running this script:
+# https://cloud.google.com/sdk/gcloud/reference/auth/login
+#
+# The default values are defined in variables.tf, do NOT modify that file.
+
+#======================= Deployment Configuration ==============================
+
+# House keeping tags. For internal Elastic projects, please follow the cloud
+# resource tagging guidelines.
 
-##house keeping tags.  Set please
 tags = {
-    "Owner" = "sunile.manjee@.co"
-    "KeepAliveUntil" = "12312023"
-    "username" = "sunman" #This can be anything you call yourself
```

Comment:
remove username will cause an issue with the automation.  I'll need to swap username with project for this to work.  Not a big issue but needs to be tested.